pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -1312,6 +1312,46 @@ func Test_server_SearchByID(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		SearchByID test cases:
+		- Equivalence Class Testing
+			- case 1.1: success search vector from 1000 vectors (type: uint8)
+			- case 1.2: success search vector from 1000 vectors (type: float32)
+			- case 2.1: fail search with non-existent ID form 1000 vectors (type: uint8)
+			- case 2.2: fail search with non-existent ID form 1000 vectors (type: float32)
+		- Boundary Value Testing
+			- case 1.1: success search with "" from 1000 vectors (type: uint8)
+			- case 1.2: success search with "" from 1000 vectors (type: float32)
```

Comment:
I am thinking do we need both uint8 and float32 cases as the test case is focus on testing the ID instead of the vector type